.github/workflows/npm-audit.yml

```diff
@@ -0,0 +1,24 @@
+name: npm audit
+
+on:
+  pull_request:
+  push:
+    branches:
+      - master
+      - 'releases/*'
+
+jobs:
+  scan:
+    name: npm audit
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: install dependencies
+        run: npm ci
+      - uses: oke-py/npm-audit-action@v1.5.0
+        with:
+          audit_level: moderate
+          github_token: ${{ secrets.GITHUB_TOKEN }}
+          # issue_assignees: oke-py // not needed
+          issue_labels: vulnerability,test
```

Comment:
I believe the convention we used for labels in other repos is `security` and I dont think we need `test` here.

Reply:
updated